client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/hooks/use-validate-verification-code.ts

```diff
@@ -62,7 +63,7 @@ export default function useValidateVerificationCode(): {
 				...( type === 'sms' && {
 					// Replace if it exists, otherwise add it
 					sms_numbers: [
-						...oldContacts.sms_numbers.filter(
+						...( oldContacts.sms_numbers || [] ).filter(
```

Comment:
`oldContacts.sms_numbers` can be undefined when the API call to fetch the contacts fails and the user adds an email contact.